### ISSUE TITLE
Skype: NVDA does not announce the 'chat content list' list box after one time for a chat window

### ISSUE THREAD

**Issue by bhavyashah** [nvaccess/nvda#5985]
Steps to reproduce
1 Open a chat window in Skype (either a contact or a group).
2 Tab over to ‘Add People button’.
3 Press Tab again.
Actual Result: NVDA says nothing.
Expected Result: NVDA should report that the focus is in ‘chat content list list’ and read the chat history under focus.

Note: Although this is consistently reproducible, this only happens once for any Skype chat window, i.e. once you reproduce this bug, if you Tab or Shift + Tab back and forth to this control, NVDA will properly announce it, but if youo close the chat window and open it again or another one, you will be able to experience this problem once.

Test Environment
Windows 8.1 Pro 64 bit
Skype™   Version 7.23.0.105 (downloaded from Skype’s official website and not the built in system app)

**Comment by Adriani90** [comment]
@bhavyashah, @florianionascu7, @k-kolev1985 can you please test again with last skype version and see if it still occurs? Thanks.

**Comment by Adriani90** [comment]
I guess the support for skype 7 is dropped anyway. Right?

cc: @josephsl 

 

Von: Kostadin Kolev <notifications@github.com> 
Gesendet: Samstag, 12. Januar 2019 17:29
An: nvaccess/nvda <nvda@noreply.github.com>
Cc: Adriani90 <adriani.botez@googlemail.com>; Comment <comment@noreply.github.com>
Betreff: Re: [nvaccess/nvda] Skype: NVDA does not announce the 'chat content list' list box after one time for a chat window (#5985)

 

This issue is relevant to Skype 7. I haven't noticed it in Skype 8 or Skype UWP.

**Comment by josephsl** [comment]
Hi, yes, Microsoft said goodbye to Skype 7. I suggest leaving this open until NVDA 2019.1 is released so folks still using Skype 7 can comment on this issue. Thanks.

**Comment by Adriani90** [comment]
As no one using Skype 7 commented on this until now, i assume most people are using Skype 8 already. I am closing this one accordingly as won't fix.
